website/docs/r/is_flow_log.html.markdown

```diff
@@ -15,47 +15,47 @@ Create, update, delete and suspend the flow log resource. For more information,
 
 ```terraform
 
-resource "ibm_is_instance" "testacc_instance" {
-  name    = "testinstance"
-  image   = "7eb4e35b-4257-56f8-d7da-326d85452591"
-  profile = "b-2x8"
+resource "ibm_is_instance" "example" {
+  name    = "example-instance"
+  image   = ibm_is_image.example.id
+  profile = "bc1-2x8"
```

Comment:
profile name modified ?

Reply:
yes.. is there any issue?

Reply:
Why is it needed?

Reply:
I feel the old one doesnt exists.. so i added the one which exists